src/reanimated2/hook/useAnimatedReaction.ts

```diff
@@ -6,6 +6,7 @@ import type { DependencyList } from './commonTypes';
 import { useSharedValue } from './useSharedValue';
 import { shouldBeUseWeb } from '../PlatformChecker';
 
+// ts-prune-ignore-next This will be used in the next PRs in this series.
```

Comment:
You can mark all your ts-prune comments with some simple keyword, so you will be able to find easily them all later and you will be sure you havn't missed anything

Reply:
It's actually reversed - I took a few changes from the mother PR and where needed I added those comments - they disappear "naturally" since they are not in the mother PR.